pages/documentation/grid/model/index.md

```diff
@@ -10,150 +10,122 @@ latex: true
+| Attribute | Description |
+| --------- | ----------- |
+| $$Country$$ | To specify in which country the substation is located |
+| $$GeographicalTags$$ | They make it possible to accurately locate the substation |
+| $$TSO$$ | To track to which [TSO](../../glossary.md#tso) the substation belongs |
 
-- [ENTSOE Area]()
+All three attributes are optional.
 
 ### Voltage Level
 [![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/VoltageLevel.html)
 
-A voltage level in IIDM represents a set of equipments connected together with the same nominal voltage, physically close to each other (~ 1-100m).
-Two voltage levels may be connected through a line (they are then located in different substations) or through transformers (they are then located within
-the same substation).
+A voltage level contains equipment with the same nominal voltage.
+Two voltage levels may be connected through lines (when they belong to different substations) or through transformers (they must be located within the same substation).
 
-A voltage level in IIDM comprises some metadata:
-- a nominal voltage (in $$kV$$)
-- a low voltage limit and a high voltage limit (in $$kV$$): they are both optional metadata. The voltage should always remain within these bounds, otherwise it means that the equipments will suffer extra wear compared to
-their normal use
-- a topology information: indicates whether the voltage level is described in [node/breaker]() or [bus/breaker]() view
-
-**Available extensions**
-
-- [Slack terminal](extensions.md#slack-terminal)
-
-#### Node/breaker topology
-<span style="color:red"> TODO: explain the topology.</span>
+| Attribute | Unit | Description |
+| --------- | ---- | ----------- |
+| $$NominalVoltage$$ | kV | Nominal base voltage |
+| $$LowVoltageLimit$$ | kV | Low voltage limit magnitude |
+| $$HighVoltageLimit$$ | kV | High voltage limit magnitude |
+| $$TopologyKind$$ |  | Level of connectivity detail |
 
-In node/breaker topology, the voltage level is described with the finest level of detail. See the sketch below for an example.
-<span style="color:red"> TODO: add sketch of voltage level.</span>
-The topology is then described as a graph structure, where busbar sections, injections or branches are connected to the vertices.
-When an equipment is connected to a vertex, in the IIDM descrition it corresponds to a `Terminal` object.
-It is not possible to connect two equipments on the same vertex.
-The edges are constituted of switches or internal connections. See the following sketch corresponding to the previous example:
-<span style="color:red"> TODO: add sketch of voltage level topology graph.</span>
+Only `NominalVoltage` and `TopologyKind` are required.
 
+The [Slack Terminal](extensions.md#slack-terminal) _extension_ defines the terminal marking which bus will be used to balance the active and reactive power in load flow analysis.
```

Comment:
Here using _italic_ for extension is not clear for me.

Reply:
You are right. I deleted it.